Documentation/security/network/encryption-wireguard.rst

```diff
@@ -293,7 +293,74 @@ options:
+|                | Service (e.g.,    |                      |                 |
+|                | NodePort)         |                      |                 |
++----------------+-------------------+----------------------+-----------------+
+| Pod            | remote Pod via    | kube-proxy           | node-to-node    |
+|                | non ClusterIP     |                      |                 |
+|                | Service           |                      |                 |
++----------------+-------------------+----------------------+-----------------+
+| Client outside | remote Pod via    | KPR,                 | default         |
+| cluster        | Service           | overlay routing,     |                 |
+|                |                   | without DSR,         |                 |
+|                |                   | without XDP          |                 |
++----------------+-------------------+----------------------+-----------------+
+| Client outside | remote Pod via    | native routing,      | node-to-node    |
+| cluster        | Service           | without XDP          |                 |
++----------------+-------------------+----------------------+-----------------+
+| Client outside | remote Pod or     | DSR in Geneve mode,  | default         |
+| cluster        | remote Node via   | without XDP          |                 |
+|                | Service           | without XDP          |                 |
++----------------+-------------------+----------------------+-----------------+
+| Pod            | remote Pod via L7 | L7 Proxy / Ingress   | default         |
+|                | Proxy or L7       |                      |                 |
+|                | Ingress Service   |                      |                 |
++----------------+-------------------+----------------------+-----------------+
+| **Egress Gateway**                                                          |
++----------------+-------------------+----------------------+-----------------+
+| Pod            |Egress Gateway node| Egress Gateway       | default         |
++----------------+-------------------+----------------------+-----------------+
+| Egress Gateway | Pod               | Egress Gateway       | default         |
+| node           |                   | without XDP          |                 |
++----------------+-------------------+----------------------+-----------------+
+
+* **Pod**: Cilium-managed K8s Pod running in non-host network namespace.
+* **Node**: K8s host running Cilium, or Pod running in host network namespace
+  managed by Cilium.
+* **Service**: K8s Service (ClusterIP, NodePort, LoadBalancer, ExternalIP).
+* **Client outside cluster**: Any client which runs outside K8s cluster.
+  Request between client and Node is not encrypted. Depending on configuration,
+  encrypted only between intermediate Node (which received client request first)
+  and destination Node.
```

Comment:
The last sentence is not clear, could you rework it please? It misses a verb, and “Depending on configuration” -> what can change based on what config?

Reply:
Something like:
> Request between client and Node is not encrypted. Depending on Cilium configuration (see the table above), it might be encrypted only between intermediate Node (which received client request first) and destination Node.

?